src/AuthKeyValueUser.php

```diff
@@ -277,4 +258,14 @@ protected function getAuthKeyValueUserService()
     {
         return ServiceManager::getServiceManager()->get(AuthKeyValueUserService::SERVICE_ID);
     }
+
+    /**
+     * @return array
+     */
+    private function getKeyValueUserData() {
+        $login = current($this->getPropertyValues(GenerisRdf::PROPERTY_USER_LOGIN));
```

Comment:
current might return false. is it ok here?

Reply:
fixed